packages/grafana-ui/src/components/DateTimePickers/TimeRangeContext.test.tsx

```diff
@@ -0,0 +1,114 @@
+import { act, render } from '@testing-library/react';
+import { useEffect } from 'react';
+
+import { makeTimeRange } from '@grafana/data';
+
+import { TimeRangeContextHookValue, TimeRangeProvider, useTimeRangeContext } from './TimeRangeContext';
+
+describe('TimeRangeProvider', () => {
+  it('provides the context with default values', () => {
+    let context: TimeRangeContextHookValue | undefined = undefined;
+    function onContextChange(val?: TimeRangeContextHookValue) {
+      context = val;
+    }
```

Comment:
I think you can move this part in `beforeEach` to avoid repeating it.

Reply:
So this is just a static declaration not an expression so putting it all into a beforeEach does not work (as the declaration would not be visible to other) I can put it into a describe block, which works, but I wonder isn't jest parallelizing the tests? Like wouldn't in that case added `await` somewhere possibly create errors?

Reply:
Hmm ok based on this https://stackoverflow.com/questions/39831961/are-tests-inside-one-file-run-in-parallel-in-jest they should not run in parallel in one file so it should be safe.